app/routes/docs/newproducers.mdx

```diff
@@ -0,0 +1,75 @@
+---
+handle:
+  breadcrumb: New Producers
+---
+
+import {
+  ProcessList,
+  ProcessListItem,
+  ProcessListHeading,
+  SummaryBox,
+} from '@trussworks/react-uswds'
+import { GitHubLabel } from '~/components/GitHubLabel'
```

Comment:
These improts are unused. If you don't want to use the process list, then please remove these imports and change the file extension to `.md`.

Reply:
I actually changed it to use the process list.  Please take a look.